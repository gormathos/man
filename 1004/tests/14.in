10
-6.60153 5.73449 9.87374 -2.81526 5.69452 -3.13478 1.56905 -8.6021 -0.522813 -7.06432
10
6.14843 5.59166 0.580499 9.22744 7.47288 6.11551 4.78328 -5.52638 9.30982 -0.058671
