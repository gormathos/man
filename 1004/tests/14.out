1.00000 5.86900 -158.16310 -998.87091 6972.97822 47465.63805 -84468.81527 -738561.36018 -365953.39033 1786304.17003 936400.85055
-4421581.92341 -248640.74233 1699335.21979 -407656926.61550 -125759716.09857 -3707763.72719 -8174361.45056 -2278110.13007 -390665615.67229 830485.01026
-22814986.98506 3998055.65607 578112.76056 163151127.71223 -177196884.72412 -20566697.33208 10959508.02568 -1218375.87876 251407985.70607 1821689.87484
